src/vm/reflectioninvocation.cpp

```diff
@@ -2857,6 +2862,7 @@ FCIMPL3(void, ReflectionInvocation::ExecuteCodeWithGuaranteedCleanup, Object* co
         // attempt to run the backout code.
         PrepareMethodDesc(g_pExecuteBackoutCodeHelperMethod, Instantiation(), Instantiation(), FALSE, TRUE);
     }
+#endif // FEATURE_CER
 
     ExecuteCodeWithGuaranteedCleanupHelper(&gc);
```

Comment:
@jkotas I think this is a good suggestion that can be implemented as a seperate issue. At minimum, it will be good to have a basic change that disables CER such that we can take it into 1.1.X branch without requiring a more involved change.